src/main.py

```diff
@@ -231,7 +231,7 @@ def run(self):
                                     sub.unsticky()
 
                                 print "Submitting game thread..."
-                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game"))
+                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game",self.TEAM_CODE))

```

Comment:
I'm working with the creator of baseball.theater. He created mappings for these dated URLs to redirect to the actual game URLs. Example for yesterday's Phillies game: http://baseball.theater/team/phi/game/20170419 redirects to https://baseball.theater/game/20170419/490320. 

I'm going to ask him to map both abbreviations for all teams, where they're different. If you feel strongly about using the more common abbreviations, I can make that change too.